sleyster_a07b/part_2/js/script.js

```diff
@@ -0,0 +1,131 @@
+
+        fetch(imageUrl)
+            .then ((response ) => {
+                if(response.ok) {
+            let image = document.createElement('img');
+            image.innerHTML = imageUrl;
+            right.appendChild(image);
+                }
+                })  
+        //End Step 3
+              
+        
+
+//Helper functions
+    function clearContainer(side){
+        switch(side){
+            case 'right':
+                right.innerHTML = '';
+                break;
+            case 'left':
+                left.innerHTML = '';
+                break;
+        }
+    }
+
+    function displayLoading(side, loadingText){
+        if(loadingText == undefined) loadingText = 'Loading content ...';
+        if((side != 'left') && (side !=' right')){
+            throw new Error ('displayLoading accepts "right" or "left"');
+        } else {
+            const container = document.getElementById(side);
+
+            let loadingTextContainer = document.createElement('p');
+            loadingTextContainer.innerHTML = loadingText;
+            container.appendChild(loadingTextContainer);
+
+            let loadingImageContainer = document.createElement('div');
+            loadingImageContainer.classList.add('loading');
+            loadingImageContainer.classList.add('centered');
+            container.appendChild(loadingImageContainer);
+        }
+    }
+
+    function onChange(evt){
+        console.log(evt.target.value);  
+    }
+//Step 2 Helper Function
+    function displayEmployeeInfo(side, fNameInfo, lNameInfo, department, salary, hireDate){
+        if ((side != 'left') && (side != 'right')){
+            throw new Error ('Error: displayEmployeeInfo accepts "right" and "left"');
+        } 
+        else {
+            const recordContainer = document.getElementById(side);
+            
+            let recordFirstNameContainer = document.createElement('h1');
+            recordFirstNameContainer.innerHTML = fNameInfo;
+            recordContainer.appendChild(recordFirstNameContainer);
+
+            let recordLastNameContainer = document.createElement('h1');
+            recordLastNameContainer.innerHTML = lNameInfo;
+            recordContainer.appendChild(recordLastNameContainer);
+
+            let recordDepartmentContainer = document.createElement('h2');
+            recordDepartmentContainer.innerHTML = department;
+            recordContainer.appendChild(recordDepartmentContainer);
+
+            let recordSalaryContainer = document.createElement('h2');
+            recordSalaryContainer.innerHTML = salary;
+            recordContainer.appendChild(recordSalaryContainer);
+
+            let recordHireDateContainer = document.createElement('h2');
+            recordHireDateContainer.innerHTML = hireDate;
+            recordContainer.appendChild(recordHireDateContainer);
+            }
+        clearContainer('right');
+        displayLoading('right', 'Loading content...');
+
+    }
+
```

Comment:
I can see that you needed to spend a bit more time on this part. You had a couple of errors early on that kept you from working through with functionality. I'll give you half credit.